tests/client/reducers/foods.tests.js

```diff
@@ -0,0 +1,9 @@
+import { GET_CATEGORY_SUCCESS } from '..//../../client/actions/category'
+import category from '../../../client/reducers/foods'
+
+describe('jest', () => {
+  it('jest is working', () => {
```

Comment:
@Rahul-Lal, you and I both know that this isn't what @emilyparkes meant when she said to include a test 😉